ibis/backends/dask/execution/window.py

```diff
@@ -84,3 +99,67 @@ def execute_window_op(
         **kwargs,
     )
     return _post_process_empty(result, data, timecontext)
+
+
+def execute_grouped_window_op(
+    op, data, window, scope, timecontext, aggcontext, clients, **kwargs,
+):
+    # extract the parent
+    (root,) = op.root_tables()
+    root_expr = root.to_expr()
+
+    data = execute(
+        root_expr,
+        scope=scope,
+        timecontext=timecontext,
+        clients=clients,
+        aggcontext=aggcontext,
+        **kwargs,
+    )
+
+    group_by = window._group_by
+    grouping_keys = [
+        key_op.name
+        if isinstance(key_op, ops.TableColumn)
+        else execute(
```

Comment:
What is the type of grouping_keys? Looks like it sometimes is list of string and sometimes is list of dd.Series?

Reply:
They're strings. 

But I actually can't find a scenario in tests that hits the else branch here. I stole this pretty directly from https://github.com/ibis-project/ibis/blob/f6910dc666fb6c950e2b386d6b9b22554adcd0c0/ibis/backends/pandas/execution/window.py#L300, so maybe this logic is outdated?

Reply:
I think 

```
else execute(key, ...)
```
will not give you back a string. (Probably give it back a dd.Series)

Reply:
If we never expect hit the else branch let's just remove it.